devito/passes/iet/misc.py

```diff
@@ -192,6 +193,28 @@ def minimize_symbols(iet):
     return iet, {}
 
 
+@iet_pass
+def complex_include(iet):
+    """
+    Add headers for complex arithmetic
+    """
+    if configuration['language'] == 'cuda':
+        lib = 'cuComplex.h'
+    elif configuration['language'] == 'hip':
+        lib = 'hip/hip_complex.h'
+    else:
+        lib = 'complex.h'
+
+    functions = FindSymbols().visit(iet)
+    for f in functions:
```

Comment:
Could be `for f in FindSymbols().visit(iet)`

Reply:
updated